release-notes/major10/10.0.0/migration.md

```diff
@@ -24,4 +24,99 @@ This page describes how to update Valtimo from the previous version to the curre
     command in the root of the implementation project:
     `ng update @angular/core@14 @angular/cli@14`
 
-       
+* **Dependency upgrades**
+
+  Scope: Frontend
+
+    In order to use the front-end libraries with upgraded dependencies, make sure to first upgrade your implementation
+    to Angular 14 by following the steps mentioned on this page.
+
+    *Redundant stylesheets*
```

Comment:
Maybe this makes more sense as an indented bullet? Same goes for Matching dependencies.